charts/csi-isilon/values.yaml

```diff
@@ -365,37 +368,51 @@ authorization:
   # skipCertificateValidation: certificate validation of the csm-authorization server
   # Allowed Values:
   #   "true" - TLS certificate verification will be skipped
-  #   "false" - TLS certificate will be verified 
-  # Default value: "true" 
+  #   "false" - TLS certificate will be verified
+  # Default value: "true"
   skipCertificateValidation: true
 
+# Storage Capacity Tracking
```

Comment:
storageCapacity: enabled :true should be added in charts/container-storage-modules/values.yaml as well

Reply:
also kindly update the same in this repository as well https://github.com/dell/csm-docs/blob/release-1.7.0/content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.7.0-values.template

Reply:
Ok